pkgs/applications/science/electronics/nxpmicro-mfgtools/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv,
+  fetchFromGitHub,
+  cmake,
+  bzip2,
+  libusb1,
+  libzip,
+  openssl,
+  pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  name    = "nxpmicro-mfgtools-1.3.191";
+  version = "uuu_1.3.191";
+
+  meta = with stdenv.lib; {
+    description = "uuu (Universal Update Utility), mfgtools 3.0";
+    longDescription = ''
+      Freescale/NXP I.MX Chip image deploy tools.
+    '';
+    homepage = https://github.com/NXPmicro/mfgtools;
+    license = licenses.bsd3;
+    maintainers = [ maintainers.bmilanov ];
+    platforms = [ "x86_64-linux" ];
```

Comment:
Are you sure it's Linux only? The project's readme says:
> The real cross platform. Linux, Windows, MacOS(not test yet)

Maybe try `platforms.all`.

Reply:
It says it's cross-platform, but I wanted to add only the platform that I am testing on.

Should I add all stated working, or should I add only tested platforms?

Reply:
I don't know what the NixOS policy exactly is on this but maybe it's fine if we put "all" and the ofborg bot doesn't complain while building for all the platforms it supports.

Reply:
13f3760 fingers crossed